src/main/java/com/ss/training/utopia/orchestrator/security/UserDetailsClass.java

```diff
@@ -0,0 +1,67 @@
+package com.ss.training.utopia.orchestrator.security;
+
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import org.springframework.security.core.GrantedAuthority;
+import org.springframework.security.core.authority.SimpleGrantedAuthority;
+import org.springframework.security.core.userdetails.UserDetails;
+
+/**
+ * @author Trevor Huis in 't Veld
+ * @author Justin O'Brien
+ */
+public class UserDetailsClass implements UserDetails {
+
+	private static final long serialVersionUID = 1557339291337891604L;
+	private String username;
+	private String password;
+	private boolean enabled;
+	private List<GrantedAuthority> authorities;
+
+	public UserDetailsClass(User user) {
+		this.username = user.getUsername();
+		this.password = user.getPassword();
+		this.enabled = user.isEnabled();
+		this.authorities = Arrays.stream(user.getRole().split(",")).map(role -> "ROLE_" + role)
+				.map(SimpleGrantedAuthority::new).collect(Collectors.toList());
+	}
+
+	@Override
+	public Collection<? extends GrantedAuthority> getAuthorities() {
+		return authorities;
+	}
+
+	@Override
+	public String getPassword() {
+		return password;
+	}
+
+	@Override
+	public String getUsername() {
+		return username;
+	}
+
+	@Override
+	public boolean isAccountNonExpired() {
+		return true;
+	}
+
+	@Override
+	public boolean isAccountNonLocked() {
+		return true;
+	}
+
+	@Override
+	public boolean isCredentialsNonExpired() {
+		return true;
+	}
+
+	@Override
+	public boolean isEnabled() {
+		return enabled;
```

Comment:
Now that I take a closer look, I think we can just make this return true, and get rid of the enabled field (for this class and the user class) altogether.

Reply:
done